packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/server/translations/Messages_fr_FR.json

```diff
@@ -331,6 +331,8 @@
   "server_configuration_installation_title": "Installation du serveur",
   "server_configuration_installation_info": "Vous êtes sur le point d'installer un OS sur le serveur <b>{{t0}}</b>. Choisissez le type d'installation que vous souhaitez utiliser :",
   "server_configuration_installation_warning": "Attention : tous les disques du serveur seront formatés lors de l’installation",
+  "server_configuration_installation_custom_image_info": "Vous pourrez installer un OS depuis une image personnalisée à l'écran suivant en choisissant \"<b>{{t0}}</b>\" dans \"<b>{{t1}}</b>\".",
```

Comment:
**nitpick:** Avoid HTML in translation files. Add the `bold` tags in the template where `t0` and `t1` is sent.
```suggestion
  "server_configuration_installation_custom_image_info": "Vous pourrez installer un OS depuis une image personnalisée à l'écran suivant en choisissant \"{{t0}}\" dans \"{{t1}}\".",
```